### DISCUSSION TITLE
Login error on mobile device

### DISCUSSION THREAD

**Discussion by nickcd** [YukiGasai/obsidian-google-calendar#147] [Q&A]
Hi! Really love the features of this plugin, however it doesn't seem to work on mobile. If I try to login on my Android I always get Error 400: redirect_uri_mismatch. I have been able to login fine on both of my desktop computers.

Just to note, I also made sure to add http://google-auth-obsidian-redirect.vercel.app and http://google-auth-obsidian-redirect.vercel.app/callback in the OAUTH client setup.

**Comment by nickcd** [comment]
So I had originally put **http** not **https** for those linked. I used **https** and now it works. Must've not copied that over from the installation instructions properly :)

**Comment by grayerbeard** [comment]
I had used htttps for those links and still did not work on mobile (Android Phone). The instruction on the set up page seem to imply different set up needed for "Andoid" but could not manage to do that. as setup then has quite dirrent requirements later on in a way not covered by the Instructions.

Abobe comment (Jul 6th) implies it should work but mine still doesnet.
